core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -114,6 +116,13 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
+  def listGroupOffsets(groupId: String): Map[TopicPartition, PartitionData] = {
+    val coordinator = findCoordinator(groupId)
+    val responseBody = send(coordinator, ApiKeys.OFFSET_FETCH, OffsetFetchRequest.allPartitionsOffsetFetchRequest(groupId))
+    val response = responseBody.asInstanceOf[OffsetFetchResponse]
+    response.responseData().asScala.toMap
```

Comment:
Thinking about this a little more.. There is an edge case around coordinator failover. We may lookup the coordinator for some group, find that it is broker A, and then send the OffsetFetch for all partitions. Before the request arrives, it could happen that broker B becomes the coordinator (it may have already begun the transition even before we did the initial coordinator lookup), but we won't have a way to detect it. This will cause us to mistakenly report that there are no offsets for the group.